comment/models/followers.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.db import models
+
+from comment.managers.followers import FollowerManager
+
+
+class Follower(models.Model):
+    email = models.EmailField()
+    username = models.CharField(max_length=50)
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    object_id = models.PositiveIntegerField()
+    content_object = GenericForeignKey()
```

Comment:
why are you adding a generic foreignkey here? If I understand correctly, this model will only be associated with a comment. Also, the email field can be `None`.

Reply:
Follower can be associated with Comment model and with Content type that uses the comment.

Reply:
okay, got it. if i have two models, i can choose to associate this with only one of them. nice, really nice.